lib/administrate/search.rb

```diff
@@ -19,9 +19,10 @@ def run
     private
 
     delegate :resource_class, to: :resolver
+    delegate :table_name, to: :resource_class
```

Comment:
kind of thinking this was a bad idea, and that using `resource_class.table_name` might be a bit more clear. let me know what you think!